lib/xcodeproj/constants.rb

```diff
@@ -104,6 +104,7 @@ module Constants
       :dynamic_library   => 'com.apple.product-type.library.dynamic',
       :static_library    => 'com.apple.product-type.library.static',
       :bundle            => 'com.apple.product-type.bundle',
+      :octest_bundle     => 'com.apple.product-type.bundle',
```

Comment:
Yup I think it is (except if I understood that stuff wrong). That's because when we use that to create a new Test target [like here](https://github.com/CocoaPods/Xcodeproj/pull/288/files#diff-dd355aa42a70484e4c3e93ebece7c06dR75), we sure want the product-type to be `com.apple.product-type.bundle`… but we can't use `:bundle` because we also want [the `wrapper_extension` of this target to be `.octest`](https://github.com/CocoaPods/Xcodeproj/pull/288/files#diff-2fbe57544f33c804869fe7b12f1728eaR125) instead of `.bundle`.